assets/js/type-defs/contexts.js

```diff
@@ -94,51 +74,43 @@
- * @property {boolean} isStarted    If true then the payment method has been
- *                                  initialized and has started.
- * @property {boolean} isProcessing If true then the payment method is
- *                                  processing payment.
- * @property {boolean} isFinished   If true then the payment method is in a
- *                                  finished state (which may mean it's status
- *                                  is either error, failed, or success)
- * @property {boolean} hasError     If true then the payment method is in an
- *                                  error state.
- * @property {boolean} hasFailed    If true then the payment method has failed
- *                                  (usually indicates a problem with the
- *                                  payment method used, not logic error)
- * @property {boolean} isSuccessful If true then the payment method has
- *                                  completed it's processing successfully.
+ * @property {boolean} isPristine   If true then the payment method state in checkout is pristine.
+ * @property {boolean} isStarted    If true then the payment method has been initialized and has
+ *                                  started.
+ * @property {boolean} isProcessing If true then the payment method is processing payment.
+ * @property {boolean} isFinished   If true then the payment method is in a finished state (which
+ *                                  may mean it's status is either error, failed, or success).
+ * @property {boolean} hasError     If true then the payment method is in an error state.
+ * @property {boolean} hasFailed    If true then the payment method has failed (usually indicates a
+ *                                  problem with the  payment method used, not logic error)
+ * @property {boolean} isSuccessful If true then the payment method has  completed it's processing
+ *                                  successfully.
  */
 
 /**
  * A saved customer payment method object (if exists)
  *
  * @typedef {Object} CustomerPaymentMethod
  *
- * @property {Object}  method     The payment method object (varies on what it
- *                                might contain)
+ * @property {Object}  method     The payment method object (varies on what it might contain)
  * @property {string}  expires    Short form of expiry for payment method.
- * @property {boolean} is_default Whether it is the default payment method of
- *                                the customer or not.
+ * @property {boolean} is_default Whether it is the default payment method of the customer or not.
  * @property {number}  tokenId    The id of the saved payment method.
- * @property {Object}  actions    Varies, actions that can be done to interact
- *                                with the payment method.
+ * @property {Object}  actions    Varies, actions that can be done to interact  with the payment
```

Comment:
Not blocking, but a double-space was introduced here.
```suggestion
 * @property {Object}  actions    Varies, actions that can be done to interact with the payment
```